codalab/client/local_bundle_client.py

```diff
@@ -167,10 +167,10 @@ def get_bundle_target(self, target):
         return (self.model.get_bundle(bundle_uuid), subpath)
 
     def get_worksheet_uuid(self, base_worksheet_uuid, worksheet_spec):
-        if worksheet_spec == '' or worksheet_spec == worksheet_util.HOME_WORKSHEET:
-            # Default worksheet name: take the username.
-            worksheet_spec = spec_util.home_worksheet(self._current_user_name())
-            return self.new_worksheet(worksheet_spec, True)
+        if worksheet_spec == '' or worksheet_spec == worksheet_util.DASHBOARD:         
```

Comment:
The landing page on the website is the dashboard, but that's taken care of
already when you have name=dashboard.

On the command-line, when you do cl work main::, it should take you to your
home directory in analog with what happens on the shell.

On Mon, Feb 8, 2016 at 7:47 PM, ChengshuLi <notifications@github.com> wrote:

> In codalab/client/local_bundle_client.py
> <https://github.com/codalab/codalab-cli/pull/333#discussion_r52264726>:
>
> > @@ -167,10 +167,10 @@ def get_bundle_target(self, target):
> >          return (self.model.get_bundle(bundle_uuid), subpath)
> >
> >      def get_worksheet_uuid(self, base_worksheet_uuid, worksheet_spec):
> > -        if worksheet_spec == '' or worksheet_spec == worksheet_util.HOME_WORKSHEET:
> > -            # Default worksheet name: take the username.
> > -            worksheet_spec = spec_util.home_worksheet(self._current_user_name())
> > -            return self.new_worksheet(worksheet_spec, True)
> > +        if worksheet_spec == '' or worksheet_spec == worksheet_util.DASHBOARD:
>
> Just wanna clarify. Do we want our landing page to be the dashboard or the
> home worksheet? I think I personally prefer the former.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codalab/codalab-cli/pull/333/files#r52264726>.
>